packages/block-library/src/home-link/index.php

```diff
@@ -138,25 +138,7 @@ function render_block_core_home_link( $attributes, $content, $block ) {
 	$html .= '>';
 
 	if ( isset( $attributes['label'] ) ) {
-		$html .= wp_kses(
-			$attributes['label'],
-			array(
-				'code'   => array(),
-				'em'     => array(),
-				'img'    => array(
-					'scale' => array(),
```

Comment:
WP Core's copy of kses doesn't include the `scale` attribute but I it's [not documented on MDN](http://developer.mozilla.org/en-US/docs/Web/HTML/Element/img) so I suspect this is a confusion with the CSS property.

Reply:
Thanks for checking that. I also couldn't see a `scale` attribute, so I guess it must have been an error.

I'll go ahead and merge this.